src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -53,5 +53,10 @@ public static class RpcWorkerConstants
         public const string DotNetExecutableName = "dotnet";
         public const string DotNetExecutableNameWithExtension = DotNetExecutableName + ".exe";
         public const string DotNetFolderName = "dotnet";
+
+        // Language worker concurrency limits
+        public const string PythonThreadpoolThreadCount = "PYTHON_THREADPOOL_THREAD_COUNT";
```

Comment:
Could you please add the link to the external documentation of both these settings - to give future context on where these things came from.